src/app/games/quarto/QuartoRules.ts

```diff
@@ -15,103 +15,100 @@ import { QuartoFailure } from './QuartoFailure';
 
 export interface BoardStatus {
     score: SCORE;
-    casesSensibles: CaseSensible[];
+    casesSensibles: SensibleSquare[];
 }
-class CaseSensible {
-    criteres: Critere[];
-    // listes des critères qu'il faut remplir dans cette case pour gagner
-    // si la pièce en main match un de ces critères, c'est une pré-victoire
+class SensibleSquare {
+    /* list of criterias that need to be fulfilled in this square in order to win
+       if the piece in hand matches one of these criterion, this is a pre-victory */
+    criteria: Criterion[];
     x: number;
     y: number;
 
     constructor(x: number, y: number) {
-        this.criteres = new Array<Critere>(3);
-        /* une case sensible peut faire au maxium partie de trois lignes
-         * l'horizontale, la verticale, la diagonale
-         */
+        /* a sensible square can be in maximum three lines
+           the horizontal, the vertical, and the diagonal */
+        this.criteria = new Array<Criterion>(3);
         this.x = x;
         this.y = y;
     }
-    public addCritere(c: Critere): boolean {
+    public addCriterion(c: Criterion): boolean {
         /* Add criterion in case several line contains this sensible case (1 to 3 lines could)
          * without duplicates
-         * return true if the criterion has been added
-         */
+         * return true if the criterion has been added */
         const i: number = this.indexOf(c);
         if (i > 0) {
-            // pas ajouté, compté en double
+            // not added, counted twice
             return false;
         }
-        assert(i !== 3, `CECI EST IMPOSSIBLE, on a rajouté trop d'éléments dans cette CaseSensible`);
-        // TODO enlever ce débug
-        this.criteres[-i - 1] = c;
+        // TODO remove this debug
+        assert(i !== 3, `This is impossible, too many elements were added in this SensibleCase`);
+        this.criteria[-i - 1] = c;
         return true;
     }
-    public indexOf(c: Critere): number {
+    public indexOf(c: Criterion): number {
```

Comment:
was singular is named as plural ?

Reply:
aaaah non, trop bizarre, ion ça sonne pluriel. J'utiliserai le "on[e]" de fin comme mnémotechnique !